tests/Application/templates/bundles/SyliusShopBundle/Cart/Summary/_totals.html.twig

```diff
@@ -0,0 +1,65 @@
+{% import "@SyliusShop/Common/Macro/money.html.twig" as money %}
+
+{% set itemsSubtotal = sylius_order_items_subtotal(cart) %}
```

Comment:
if You are going to change variables in .twig to snake case - do it here as well items_Subtotal